spec/models/form_profile_spec.rb

```diff
@@ -483,9 +508,10 @@ def expect_prefilled(form_id)
     end
 
     context 'with emis data', skip_emis: true do
-      before do
+      # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+      def stub_methods_for_emis_data(military_branch = 'air force')
```

Comment:
Refactored this from a `before` to it's own method to be called by within different scope/context just because the Feedback Tool's schema expects capitalized (e.g. 'Air Force') instead of lower case (e.g. 'air force') military branches.  I tried just re-declaring the `expect(military_information).to receive(:last_service_branch).and_return('Air Force')` within my specific `complaint tool` spec, but it just does not work.